tests/conftest.py

```diff
@@ -19,3 +32,489 @@ def environment_url(request):
     SQS_STAGE: SERVICE_SQS_STAGE,
     SQS_PROD: SERVICE_SQS_PROD
 }
+
+# Numia pool type constants using an Enum
+class NumiaPoolType(Enum):
```

Comment:
Note to reviewer: most of the diff in this file was moved from `setup.py` due to refactoring setup logic and having to rely on the built-in functionality of `conftest.py`.

I will mark the new areas of the codebase with comments